fractor-xml/src/XmlFileProcessor.php

```diff
@@ -41,6 +42,8 @@ public function handle(File $file): void
         $iterator->traverseDocument($document);
 
         $newFileContent = $this->saveXml($document);
+        // TODO make the indentation configurable in fractor config
```

Comment:
How can we make this configurable in fractor config?

Reply:
@sabbelasichon How can we make this configurable in fractor config?

Reply:
I have created two new options and the generic method to configure those options in the FractorConfiguration. I think we should not bind the fractor base with processor specific configurations hence the generic option. Could you please add this to the README.md